src/main/java/com/redhat/cloud/notifications/routers/ApplicationService.java

```diff
@@ -17,7 +18,7 @@
 import javax.ws.rs.core.MediaType;
 import java.util.UUID;
 
-@Path("/applications")
+@Path(Constants.API_INTEGRATIONS_V_1_0 + "/applications")
```

Comment:
``/applications`` is internal, it should not be exposed to 3scale (the UI doesn't use /applications). So is this path meaningful?